components/prism-vim.js

```diff
@@ -0,0 +1,9 @@
+	'number': /\b-?(0x)?\d*\.?\d+\b/,
+	'operator': /[-+]{1,2}|!|=?&lt;|=?&gt;|={1,2}|(&amp;){1,2}|\|?\||\?|\*|\//,
```

Comment:
`[-+]{1,2}` matches `-+` and `+-`. You probably wanted to write `--?|\+\+?`
Also, the HTML-escaped are not used anymore in Prism, please use `<` `>` and `&` directly.
Finally, you might want to combine those single chars (`!` `?` `*` and `/`) together in a character class: `[!?*\/]`